src/libraries/System.Runtime.InteropServices.JavaScript/src/Resources/Strings.resx

```diff
@@ -129,6 +129,21 @@
   <data name="TypedArrayNotCorrectType" xml:space="preserve">
     <value>TypedArray is not of correct type.</value>
   </data>
+  <data name="MissingEntrypoint" xml:space="preserve">
+    <value>Missing entrypoint</value>
+  </data>
+  <data name="EntrypointHandle" xml:space="preserve">
```

Comment:
Maybe `CannotResolveManagedEntrypointHandle`, and `Cannot resolve managed entrypoint handle`